src/test/java/com/cst438/controller/SectionControllerUnitTest.java

```diff
@@ -103,6 +104,7 @@ public void addSectionFailsBadCourse( ) throws Exception {
                 2024,
                 "Spring",
                 "cst599",
+                "Title",
```

Comment:
I know I asked you to remove my changes, but I think these here are needed for these unit tests to work.